docs/howto/outside_controller_use.md

```diff
@@ -0,0 +1,37 @@
+## Using AMS Outside Of A Controller
+
+### Serializing a resource 
+
+In the current version of AMS, serializing resources outside controllers is
+fairly simple:
+
+```ruby
+# Create our resource
+post = Post.create(title: "Sample post", body: "I love Active Model Serializers!")
+
+# Optional options parameters
+options = {}
+
+# Create the serializer
+serializer = ActiveModel::SerializableResource.serialize(post, options)
```

Comment:
There's a terminology issue here. It is not specific to this PR, but `ActiveModel::SerializableResource.serialize()` actually returns an instance of `ActiveModel::SerializableResource` (it is equivalent to `ActiveModel::SerializableResource.new()`.
Hence, a possible replacement would be:
```ruby
serializable_resource = ActiveModel::SerializableResource.new(post, options)
model_json = serializable_resource.as_json
```